### PR TITLE
Creating keycloak workshop

### PR DESCRIPTION
**Pull request by mcruzdev**
This PR is from #1 issue

I am creating this PR for you to see my changes and to work together
You are free to give me a suggestion and improvement

### CODE REVIEW

**Comment by salaboy** [comment]
@mcruzdev concentrate on adding the steps to reproduce your setup.. I can work on adding the text to explain what each step is doing.. 
Does that work for you?

**Comment by salaboy** [comment]
@mcruzdev let me know how can I help you to move forward with this.. it will be great if you just create section titles and add the steps.. I can add more text if needed.. I don't want you to worry too much about writing.

**Comment by salaboy** [comment]
@mcruzdev man.. this is really good.. if you can add the instructions on what needs to be modified in the gateway I can give it a try right now..

**Comment by salaboy** [comment]
@mcruzdev just as a reference.. -> https://www.baeldung.com/spring-oauth-login-webflux

**Comment by mcruzdev** [comment]
@salaboy,

I need to add some concepts about OAuth2, Keycloak and Spring OAuth2

File: sso.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+
+```
+package com.salaboy.conferences.c4p.rest.configuration;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.config.CorsRegistry;
+import org.springframework.web.reactive.config.WebFluxConfigurer;
+
+@Configuration
+public class CORSConfig implements WebFluxConfigurer {
```
Comment (salaboy) [inline sso.md:259]: @mcruzdev do we need CORS?

**Comment by salaboy** [comment]
@mcruzdev this is awesome! thanks a lot for that.. I have quite a busy week, but now that tracing is almost done I will move to review all these PRs.. I will probably merge the ones in the service to give it a try and then revert back . I will keep you posted if I find issues with it.

File: sso.md (hunk 1 above)
Comment (mcruzdev) [reply sso.md:259]: Hello, @salaboy, Yes, once I tested without CORS and I received an error

Comment (salaboy) [reply sso.md:259]: @mcruzdev ok.. I will check it out...

**Pull request merged**